autoload/gina/command/_raw.vim

```diff
@@ -22,6 +29,47 @@ function! s:build_args(git, args) abort
   return args.lock()
 endfunction
 
+function! s:fetch_complete(arglead, cmdline, cursorpos) abort
+  let args = gina#core#args#new(matchstr(a:cmdline, '^.*\ze .*'))
+  if empty(args.get(2))
+    return gina#complete#common#remote(a:arglead, a:cmdline, a:cursorpos)
+  endif
+  if empty(args.get(3))
+    " TODO: Return refspecs in remote repository "args.get(2)".
```

Comment:
The refspec in the remote repository `args.get(2)` should be returned here, but I couldn't think of a good way to implement it, so I made it a TODO.